app/src/main/AndroidManifest.xml

```diff
@@ -19,7 +19,8 @@
         android:icon="@drawable/launcher_image"
         android:label="@string/app_name"
         android:supportsRtl="true"
-        android:theme="@style/AppTheme">
+        android:theme="@style/AppTheme"
+        android:usesCleartextTraffic="true">
```

Comment:
You have made this change in this [PR](https://github.com/apache/fineract-cn-mobile/pull/75/commits/7c7149dbbfd9866925c3036d70543cae0d66584f) and it is not relevant to this PRs too, so please remove it.

Reply:
I've removed them in the next commit.